src/Resources/Guests/UndeliverableDate.ts

```diff
@@ -0,0 +1,14 @@
+import AbstractResource from '../AbstractResource';
+import { HttpClientConfig, HttpClientResponse } from '../../Interfaces/HttpClient';
+
+export default class UndeliverableDate extends AbstractResource {
+    initialise(): void {
+        this.resourcePath = '/undeliverable-dates';
+        this.defaultAccess = 'guest';
+    }
+
+    public get: (...args: HttpClientConfig) => HttpClientResponse = this.createMethodFromPartialSpec({
+        method: 'GET',
+        path: '',
+    });
```

Comment:
`get` implies we're getting something specific. I think you may want `all` here instead